api_documentation_v1.yaml

```diff
@@ -0,0 +1,406 @@
+      title: LoadingDetailsExternalDTO
+      type: object
+      properties:
+        endDate:
+          type: string
+          description: Loading finish date and time (UTC time)
+          example: '2020-11-06T09:35:53Z'
+        ingredients:
+          type: array
+          description: Loaded ingredients in mixer wagon
+          items:
+            $ref: '#/components/schemas/FeedingLoadIngredientExternalDTO'
+        mixerWagon:
+          description: Mixer wagon used for loading
+          $ref: '#/components/schemas/MixerWagonExternalDTO'
+        startDate:
+          type: string
+          description: Loading start date and time (UTC time)
+          example: '2020-11-06T08:35:53Z'
+    MixerWagonExternalDTO:
+      title: MixerWagonExternalDTO
+      type: object
+      properties:
+        id:
+          type: integer
+          description: Mixer wagon ID
+          format: int64
+          example: 478
+        imageUrl:
+          type: string
+          description: Mixer wagon image url
+        name:
+          type: string
+          description: Mixer wagon name
+          example: USA x 15
+        weight:
+          type: number
+          description: Loading start date and time (unit Tonnes)
+          format: double
+          example: 2
+    PenExternalDTO:
+      title: PenExternalDTO
+      type: object
+      properties:
+        feedIntakePerAnimalKg:
+          type: number
+          description: Feed intake per animal (unit KG)
+          format: double
+        id:
+          type: integer
+          description: Pen ID
+          format: int64
+          example: 12
+        numberOfAnimals:
+          type: integer
+          description: Number of animals in pen
+          format: int32
+          example: 100
+        penName:
+          type: string
+          description: Pen name
+          example: Test pen
+    RecipeDailyReportExternalDTO:
+      title: RecipeDailyReportExternalDTO
+      type: object
+      properties:
+        feedingDate:
+          type: string
+          description: 'Feeding date '
+          example: '2020-11-05'
+        feedings:
+          type: array
+          description: Daily feedings for recipe
+          items:
+            $ref: '#/components/schemas/FeedingReportExternalDTO'
+        herdId:
+          type: integer
+          description: Herd ID
+          format: int64
+          example: 245
+        rationExternalId:
+          type: integer
+          description: ExternalId represent identification number of ration between systems
+          format: int64
+          example: 41933
+        rationName:
+          type: string
+          description: Ration name
+          example: Test ration
+        recipeId:
+          type: integer
+          description: Recipe ID
+          format: int64
+          example: 421
+        recipeName:
+          type: string
+          description: Recipe name
+          example: Test recipe
+        teamId:
```

Comment:
as we don't have the concept of a team, herd, recipe in the atlas-template somewhere else, we should remove that fields here.